src/components/03-components/15-stepindicator/stepindicator.njk

```diff
@@ -1,19 +1,19 @@
 <div class="usa-step-indicator" aria-label="progress">
   <ol class="usa-step-indicator__segments">
     <li class="usa-step-indicator__segment usa-step-indicator__segment--complete">
-      <span class="usa-step-indicator__segment-label">Personal information <span class="usa-sr-only">completed</span></span>
+      <a class="usa-step-indicator__segment-label">Personal information <span class="usa-sr-only">completed</span></a>
```

Comment:
Looking at federalist build, despite being an anchor tag, there is no indication that this item is clickable - I think you also need an href to get that behavior. Alternatively, uswds has a `cursor-pointer` class that turns the cursor to the little hand icon indicating click-ability on hover. This class is what I used in the angular version of the step indicator - Looking at the federalist build, it seems like despite having an anchor tag, the items don't indicate they are inter actable - something like a cursor change / highlight change etc. Maybe you also need href in anchor tags to get the interactive experience. Also would we want just the text to be clickable or the step segments as well? I know you weren't able to stay for the entire demo last Friday, but let me know what's your take in terms of interactivity on the angular step indicator - https://bibek-regmi.github.io/ngx-uswds/step-indicator
Additionally, it looks like this makes just the text in each step clickable. Do we also want to make each segments clickable too? USWDS has step indicator versions where you can hide labels, and if the screen size is small enough, the labels also hide automatically for responsiveness. In those cases, users would not be able to click on specific step because the labels to click on wouldn't exist.